modules/charts/package.json

```diff
@@ -1,26 +1,30 @@
 {
 	"name": "@overture-stack/arranger-charts",
-	"private": true,
 	"version": "0.0.0-dev",
-	"type": "module",
 	"main": ".dist/main.js",
 	"module": "./dist/main.js",
 	"files": [
 		"./dist"
 	],
+	"publishConfig": {
+		"access": "public"
+	},
 	"scripts": {
-		"dev": "vite build --watch",
 		"build": "vite build",
-		"test": "vitest",
+		"build:storybook": "storybook build",
+		"dev": "vite build --watch",
 		"lint": "eslint .",
+		"nuke:dist": "rm -rf dist",
 		"storybook": "storybook dev -p 6006",
-		"build-storybook": "storybook build"
+		"test": "vitest"
 	},
 	"peerDependencies": {
 		"@emotion/react": "^11.0.0",
-		"@overture-stack/arranger-components": "^3.0.0-rc.1",
-		"react": "^18.3.1",
-		"react-dom": "^18.3.1"
+		"@overture-stack/arranger-components": "*",
```

Comment:
This is pointing to the workspace but is invalid as an actual peer dep I think?
Would work in devDep? Because we want a version for development, pointing to local workspace is fine.